### PR TITLE
WIP: Compendium of Forgotten Secrets - Part Two

### PR DESCRIPTION
**Pull request by ColdAnkles**
Adding more content from the Compendium of Forgotten Secrets

### CODE REVIEW

File: third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		</description>
+		<sheet>
+			As an action, choose any number of creatures in a 15-foot radius of you. The targets must make a Charisma saving throw against your warlock spell save DC. If they fail, they fall prone and remain prone on their next turn unless they use all of their movement to stand. Once you use this feature, you can’t do so again until you finish a short or long rest.
+		</sheet>
+	</element>
+
+	<element name="Breath of the Gale" type="Archetype Feature" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_ARCHETYPE_FEATURE_STORMLORD_BREATHOFTHEGALE">
+		<requirements />
+		<description>
+			<p>At 6th level, you passively manifest a small tempest around yourself. You gain advantage on saving throws against dangerous gases, clouds, forceful winds, and other similar effects. Your vision is not obscured by fog, mist, smoke, rain, clouds, or other airborne particles.<p>
```
Comment (LordSionnach) [inline third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml]: Unclosed  <p>.

File: third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		</description>
+		<sheet>
+			When you deal lightning or thunder damage using a spell of 1st level or higher or cast a spell that is on your patron’s extended spell list, you can use your bonus action to teleport to an unoccupied space adjacent to a target of the spell or to an unoccupied space within the spell’s area of effect.
+		</sheet>
+	</element>
+
+	<element name="Slayer of the King" type="Archetype Feature" source="Player’s Handbook" id="ID_GFP_COFSA_ARCHETYPE_FEATURE_STORMLORD_SLAYEROFTHEKING">
+		<description>
+			<p>At 14th level, you discover the unspoken secrets of the heroes who sought the downfall of the Storm Lord. As an action, you cleave the air with a blade of purest wind in a line 300 feet long, 300 feet high, and 5 feet wide. Each creature in the line must make a Dexterity saving throw against your warlock spell save DC. A creature takes force damage equal to three times your warlock level on a failed save or half as much damage on a successful one. Unattended objects automatically fail this saving throw.<p>
```
Comment (LordSionnach) [inline third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml]: P not closed.

File: third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            <p class="indent">Once you use this feature, you can’t do so again until you finish a long rest.</p>
+		</description>
+		<sheet>
+			As an action, you cleave the air with a blade of purest wind in a line 300 feet long, 300 feet high, and 5 feet wide. Each creature in the line must make a Dexterity saving throw against your warlock spell save DC. A creature takes {{slayer of the king:dmg}} force damage or half as much damage on a successful one. Unattended objects automatically fail this saving throw. Once you use this feature, you can’t do so again until you finish a long rest.
+		</sheet>
+        <rules>
+            <stat name="slayer of the king:dmg" value="level:warlock" >
```
Comment (LordSionnach) [inline third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml]: Supposedly this line -> line 142 are missing a closing /, as it outputs an error and in other patrons the "stat" line has a /, but when I added them Aurora refused to load and doesn't spit out an error. No idea what happened there.

**Comment by swdriessen** [comment]
Careful when adding files people submit without testing them, i went over them a bit, clean some things that stood out, and files should load without errors now.

If you think the content is okay you can add the files to the index and ill merge the current progress. 

When creating a new pull request you can also mark it as a draft so i'll know not to check things before you think are ready to be merged. (https://github.blog/2019-02-14-introducing-draft-pull-requests/)

In the next version sources like this can be marked in the builder like so:

image

image

You'll also be able to do this for example on your lost vault source by adding a few setters with details and an optional url can be set to for example a github issue with a checklist of items.

**Comment by ColdAnkles** [comment]
Somehow broke everything....

**Comment by swdriessen** [comment]
I assume you mean the merge commit from my changes, that you didn't expect to see?

**Comment by swdriessen** [comment]
I'll merge it and increment the version of the index if all loads well.

**Comment by swdriessen** [comment]
Thanks!

**Comment by ColdAnkles** [comment]
> 
> 
> I assume you mean the merge commit from my changes, that you didn't expect to see?

yes, lots of unrelated stuff showed up, then panic

**Comment by swdriessen** [comment]
I'll create the remaining 6 archetypes.

**Pull request merged**